curriculum/challenges/english/01-responsive-web-design/basic-css/size-your-images.english.md

```diff
@@ -34,8 +33,8 @@ Create a class called <code>smaller-image</code> and use it to resize the image
 tests:
   - text: Your <code>img</code> element should have the class <code>smaller-image</code>.
     testString: assert($("img[src='https://bit.ly/fcc-relaxing-cat']").attr('class') === "smaller-image");
-  - text: Your image should be 100 pixels wide. Browser zoom should be at 100%.
-    testString: assert($("img").width() === 100);
+  - text: Your image should be 100 pixels wide.
+    testString: assert($("img").width() < 200 && code.match(/(?<=<style>([\s\S])*)\.smaller-image[\s\n]*{([\s\S]*;)?[\s\n]*width\s*:\s*100px\s*;(?=[\s\S]*<\/style>)/i));
```

Comment:
If the user write the following, the test should not fail.
```css
.smaller-image{
  width: 100px
}
```

Reply:
Should CSS tests not fail, if expressions are missing a semi-colon at the end?

Reply:
@Sky020 Only if it would end up being invalid CSS.  Since there is only one property/value pair, this totally valid.  I will need to go back through some older PRs of mine.  You just need to modify the regex, so to handle the case where there is only one property/value pair or if there are more than one property/value pairs and `width: 100px` is the last one, then make the semi-colon optional.